modules/ModuleBuilder/parsers/parser.label.php

```diff
@@ -246,7 +240,7 @@ public static function addLabels($language, $labels, $moduleName, $basepath = nu
         if ($changed) {
             $GLOBALS [ 'log' ]->debug("ParserLabel::addLabels: writing new mod_strings to $filename");
             $GLOBALS [ 'log' ]->debug('ParserLabel::addLabels: mod_strings='.print_r($mod_strings, true));
-            if (!write_array_to_file('mod_strings', $mod_strings, $filename)) {
+            if (!write_override_label_to_file('mod_strings', $mod_strings, $filename)) {
```

Comment:
The ```write_override_label_to_file``` is pretty much identical to ```write_array_to_file```. They should be combined so we don't have duplicated code.